modules/ROOT/pages/index.adoc

```diff
@@ -29,18 +33,41 @@ You can monitor your APIs' conformance and notify developers to help improve the
 
 include::_partials/api-gov-console-scap-callouts.adoc[]
 
-== Governance in Exchange and Design Center
+== Governance in Exchange, Design Center, and API Manager
+
+API Governance is integrated with <<exchange-overview>>, <<design-center-overview>>, and <<api-manager-overview>>.
+
+[[exchange-overview]]
+=== Exchange
+
+In Exchange:
+
+* Developers can view published API specifications' conformance status details, discover provided rulesets, and publish custom rulesets. 
```

Comment:
```suggestion
* Developers can view published API specification conformance status details, discover provided rulesets, and publish custom rulesets. 
```

Reply:
(I think you don't need the possessive.)

Reply:
The conformance status details are in a separate page for each API, so I think it's a more accurate sentence with possessive. Not using it could imply that the details are in the specifications themselves. I'll give it some thought. This isn't new text.

Reply:
I reworded this a bit